website/docs/verifiable-document/comparison.md

```diff
@@ -39,9 +39,11 @@ For DID, it's the opposite, everything is private.
 
 Revocation is part of our Ethereum Smart Contracts.
 
-If you use DID, there is for the moment no way to revoke a document. We have ideas on [how to provide such a feature](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly), but the work is still in progress. In any case,
+We have ideas on [how to provide such a feature](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly), but the work is still in progress. In any case,
```

Comment:
You can get rid of this part of it's not relevant anymore

However it must be added that it you use revocation then did is not free anymore, need to pay for the deployment of revocation store